modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2463,6 +2452,28 @@ export class Wallet implements IWallet {
           ],
         });
       }
+
+      case 'Digital Asset': {
+        this.validateNftEntries(sendNftOptions.entries, nftBalance, sendNftOptions);
+
+        const recipients = _.map(sendNftOptions.entries, (entry) => {
```

Comment:
```suggestion
        const recipients = sendNftOptions.entries.map( (entry) => {
```

Reply:
also what does entries mean here ?

Reply:
entries: https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts#L457
it denotes a list of NFT IDs and their quantity for ERC1155 type NFTs.